pkg/api/install/install.go

```diff
@@ -19,4 +22,458 @@ import (
 	_ "github.com/openshift/origin/pkg/security/api/install"
 	_ "github.com/openshift/origin/pkg/template/api/install"
 	_ "github.com/openshift/origin/pkg/user/api/install"
+
+	kv1 "k8s.io/kubernetes/pkg/api/v1"
```

Comment:
[test]

On Thu, Jun 16, 2016 at 12:11 PM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> I tried to pick the same pattern so I can cut and paste (kXapi or kXv1)
>
> On Thu, Jun 16, 2016 at 10:32 AM, Michal Fojtik <notifications@github.com>
> wrote:
>
>> In pkg/api/install/install.go
>> <https://github.com/openshift/origin/pull/9365#discussion_r67355424>:
>>
>> > @@ -19,4 +22,458 @@ import (
>> >  	_ "github.com/openshift/origin/pkg/security/api/install"
>> >  	_ "github.com/openshift/origin/pkg/template/api/install"
>> >  	_ "github.com/openshift/origin/pkg/user/api/install"
>> > +
>> > +	kv1 "k8s.io/kubernetes/pkg/api/v1"
>>
>> kapiv1 ?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/openshift/origin/pull/9365/files/0768a8081118df921fba9b5cf1edfca0bac78367#r67355424>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ABG_p8lbvV0AaP_0Cs1-2i21WlAbx9vQks5qMV6JgaJpZM4I27dv>
>> .
>>
>
>